### PR TITLE
Fixed broken links and corrected domain name

### PR DESCRIPTION
**Pull request by kamn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ Inspired by the [awesome](#more-awesome) list thing. Feel free to <a href="https
 * [Pokelmon](https://github.com/brenopanzolini/pokelmon) - Elm project consuming PokéAPI.
 * [JWT auth with Django + Elm](https://github.com/apirobot/django-elm-auth-with-jwt) - JSON Web Token (JWT) authentication using Django (backend) and Elm (frontend).
 * [Bitcoin BR Chrome Extension](https://github.com/jouderianjr/bitcoin-br-chrome-extension) - Chrome extension built in Elm that shows the bitcoin value in all Brazilian exchanges.
-* [Elmstagram](https://github.com/bkbooth/Elmstagram) - Basic UI Clone of Instagram / Articles - [Part 1](https://benkbooth.com/building-a-basic-ui-clone-of-instagram-using-elm-part-1/) [Part 2](https://benkbooth.com/post/building-a-basic-ui-clone-of-instagram-using-elm-part-2/) [Part 3](https://benkbooth.com/post/building-a-basic-ui-clone-of-instagram-using-elm-part-3/)
+* [Elmstagram](https://github.com/bkbooth/Elmstagram) - Basic UI Clone of Instagram / Articles - [Part 1](https://benbooth.dev/building-a-basic-ui-clone-of-instagram-using-elm-part-1/) [Part 2](https://benbooth.dev/building-a-basic-ui-clone-of-instagram-using-elm-part-2/) [Part 3](https://benbooth.dev/building-a-basic-ui-clone-of-instagram-using-elm-part-3/)
```
Comment (nelsonic) [inline README.md:60]: The current link https://benkbooth.com/building-a-basic-ui-clone-of-instagram-using-elm-part-1 appears to re-direct to https://benbooth.dev/building-a-basic-ui-clone-of-instagram-using-elm-part-1
Might as well update the link. 
image

👍

Comment (kamn) [reply README.md:60]: The part 1 link worked but parts 2 and 3 were broken.
https://benkbooth.com/post/building-a-basic-ui-clone-of-instagram-using-elm-part-2/

Comment (nelsonic) [reply README.md:60]: Confirmed. ✅ 
https://benbooth.dev/post/building-a-basic-ui-clone-of-instagram-using-elm-part-2
image

New link: https://benbooth.dev/building-a-basic-ui-clone-of-instagram-using-elm-part-2
image


https://benbooth.dev/post/building-a-basic-ui-clone-of-instagram-using-elm-part-3
image

New link: https://benbooth.dev/building-a-basic-ui-clone-of-instagram-using-elm-part-3
image

Good PR. 👍

**Comment by sporto** [comment]
Thanks

**Pull request merged**